requirements.txt

```diff
@@ -1,14 +1,18 @@
-Flask~=1.1.2
-Flask-Classful~=0.14.2
-flask-paginate~=0.7.0
-Sphinx~=3.2.1
-sphinx_rtd_theme~=0.5.0
-sphinxcontrib-httpdomain~=1.7.0
-celery~=4.4.7
-pymongo~=3.11.0
-python-magic~=0.4.18
-requests~=2.24.0
-Flask-Login==0.5.0
-zxcvbn==4.4.28
-GitPython~=3.1.7
-markdown2~=2.3.9
```

Comment:
I changed this. version are now locked again.

I wanted to unlock the dependencies because I thought having "up to date" packages was more important than preventing stability issues. From your comment, I understand that you disagree 😅